frontend/front/src/api/apiSlice.js

```diff
@@ -67,8 +67,16 @@ export const apiSlice = createApi({
         ...result.map(({ id }) => ({ type: "Surveyor", id })),
       ],
     }),
+    // getSurveyorsByAssignmentId: builder.query({
```

Comment:
Either uncomment this (even if we don't use it now, it might be useful later) or comment the `useGetSurveyorsByAssignmentIdQuery` below (or just remove both if you don't think they're necessary)